doc/whats_new/v1.1.rst

```diff
@@ -186,6 +186,20 @@ Changelog
   hole; when set to True, it returns the swiss-hole dataset. :pr:`21482` by
   :user:`Sebastian Pujalte <pujaltes>`.
 
+- |Enhancement| Adds optional arguments `n_retries` and `delay` to functions 
+  :func:`fetch_20newsgroups`
```

Comment:
Commas between each function:

```suggestion
  :func:`fetch_20newsgroups`, 
```

This comment also applies to the rest of the functions